tagsdoc/src/main/resources/com/sun/tlddoc/resources/help-doc.html.xsl

```diff
@@ -222,6 +222,8 @@
           <HR/>
           <small><i>
             Copyright &#169; 2018, 2022 Eclipse Foundation. All rights reserved.
+            Use is subject to
+            <a href="../EFSL.html"> license terms</a>
```

Comment:
We should have a `.` (period) at the end of the license terms sentence. In addition lets have this be two lines. This would give us the same look and feel as the javadoc:  https://github.com/eclipse-ee4j/jstl-api/blob/master/api/pom.xml#L292

Reply:
Updated. Let me know what you think.

Reply:
Looks good to me! Thanks!